aten/src/ATen/native/native_functions.yaml

```diff
@@ -1197,6 +1197,9 @@
     CPU: floor_out
     CUDA: floor_out
 
+- func: floor_divide(Tensor self, Tensor other) -> Tensor
+  variants: function, method
```

Comment:
if we follow numpy, this is actually only a function.  I'd have a slight preference to do it that way because we can always add the method later.